foundations/installations/prerequisites.md

```diff
@@ -111,18 +111,19 @@ While your VM is running, do the following steps:
   3. Install all available updates. If there are no available updates, move on to Step 5.
   4. If the **Software Updater** is stuck waiting for an **unattended upgrade** to finish, reboot the VM and start again from Step 1.
   5. Open a terminal with `ctrl + alt + t` or opening the **Whisker Menu** and typing in **Terminal** (the shortcut is obviously faster).
-  6. Copy and paste this into the terminal: `sudo apt install linux-headers-$(uname -r) build-essential dkms`. Enter your password when it asks you to. _(__note__: Your password will not be visible in the terminal. This is a security feature to protect your password. Press `Enter` when done.)_
-  7. If you get the following errors: **Unable to locate package build-essential** and **Unable to locate package dkms**, paste in the following: `sudo apt-get install build-essential` and enter your password. Otherwise, move on to Step 8.
-  8. Type `Y` when it asks you to and let it finish installing. Close the terminal when it is finished.
-  9. Click **Devices** on the VM toolbar -> **Insert Guest additions CD image** in the menu bar.
-  10. Wait for the CD image to mount, it will show the CD on the desktop as solid, not transparent, and a window will show on the top right of the VM screen saying it was successfully mounted.
-  11. If you see a File Manager window appear, then confirm the presence of a file named `VBoxLinuxAdditions.run` before proceeding to step 12. But if you do _not_ see a File Manager window appear, then navigate to the desktop by minimizing all opened windows, and then double-click on the CD icon on the VM desktop.
-  12. In the new window that opens, right click on the white-space or any file/folder, and click **Open Terminal Here**.
-  13. In the newly opened terminal window, paste `sudo ./VBoxLinuxAdditions.run` and hit enter.
-  14. Once it finishes, close the terminal and the CD folder.
-  15. Right-click CD on the VM desktop and click **Eject Volume**. It will not eject if the CD folder is open.
-  16. Reboot your VM (which you can do by typing `reboot` and hitting enter in a terminal).
-  17. You can now maximize the VM window, create additional displays, and use many other useful features. These options are available on the VM toolbar under **View** and **Device**.
+  6. Copy and paste this into the terminal: `sudo apt install linux-headers-$(uname -r) build-essential dkms`. _(__note__: You cannot copy and paste to and from the guest OS and your host OS (Windows), so when copying these commands you need to open TOP in your VM through your `Web Browser` by opening the **Whisker Menu** and typing in **Web Browser**.)_
```

Comment:
```suggestion
  6. Copy and paste this into the terminal: `sudo apt install linux-headers-$(uname -r) build-essential dkms`. _(__note__: You cannot copy and paste between your guest OS and host OS (Windows), so when copying these commands you need to open this page in your VM through your `Web Browser` by opening the **Whisker Menu** and typing in **Web Browser**.)_
```
See the above suggested edit. Additionally I think on Xubuntu `ctrl + alt + w` will open a web browser, if you're currently using Xubuntu and can confirm that I would add that in as an alternative to going through the whisker menu like in step 5.

Reply:
Hi wise-king-sullyman, your edits are great; they pinpointed my exact weaknesses. I was having trouble writing that sentence and I wasn't able to find a keyboard shortcut for opening a web browser. Unfortunately, when I tried out `ctrl + alt + w` on Xubuntu nothing happened, so I suppose my instructions can remain the same.
Thanks for the help!

Reply:
It's listed as `Super + W`, which is probably `ctrl + shift + alt + W`.

Reply:
Hi xandora, unfortunately `ctrl + shift + alt + w` did not seem to do anything when I tested it out on Xubuntu, although it seemed quite plausible. Thanks for the help anyways!

Reply:
Oh, it's `Win + W`, at least on my Xubuntu VM.

Reply:
aand yep that worked

Reply:
Would only people with Windows install a Virtual Machine or do other OS's also need a Virtual Machine?

Reply:
I believe that we only instruct people to use a VM if they're currently on Windows.

Reply:
Thanks, I wasn't sure about that, I'll add `Win + W` to the instructions.